account_multicurrency_revaluation/model/res_company.py

```diff
@@ -76,13 +76,13 @@ class ResCompany(models.Model):
     reversable_revaluations = fields.Boolean(
         string="Flag Revaluation Entries",
         help="If marked, the entries created in the process will be flaged "
-        "to be able to filter them out later.",
+        "to be able to filter them out later. ",
         default=True,
     )
     auto_post_entries = fields.Boolean(
         string="Auto Post Created Entries",
         help="If marked, the entries created in the process will be"
-        "posted automatically.",
+        "posted automatically. ",
```

Comment:
The space is needed to separate `be` and `posted` so below is the suggested change.
```suggestion
        " posted automatically.",
```